jekyll/_cci2/configuration-reference.md

```diff
@@ -27,12 +27,62 @@ You can see a complete `config.yml` in our [full example](#full-example).
 
 Key | Required | Type | Description
 ----|-----------|------|------------
-version | Y | String | `2`, `2.0`, or `2.1` See the Reusing Config doc for an overview of new 2.1 keys available to simplify your `.circleci/config.yml` file, reuse, and parameterized jobs.
+version | Y | String | `2`, `2.0`, or `2.1` See the [Reusing Config]({{ site.baseurl }}/2.0/reusing-config/) doc for an overview of new 2.1 keys available to simplify your `.circleci/config.yml` file, reuse, and parameterized jobs.
 {: class="table table-striped"}
 
 The `version` field is intended to be used in order to issue warnings for deprecation or breaking changes.
 
-## **`commands`** (requires version:2.1)
+## **`orbs`** (requires version: 2.1)
+
+Key | Required | Type | Description
+----|-----------|------|------------
+orbs | N | String | The name of an orb to use.
+executors | N | String | The name of an executor to declare.
+commands | N | String | The name of a command to declare.
+jobs | N | String | The name of a job to declare.
+{: class="table table-striped"}
+
+The following example calls an Orb named `hello-build` that exists in the certified `circleci` namespace.
+
+```
+version: 2.1
+orbs:
+    hello: circleci/hello-build@volatile
```

Comment:
We should not use `volatile` in the main sample of orbs. The `volatile` version is an advanced feature that should not be used unless the user is aware of the consequences. Let's replace `circleci/hello-build@volatile` with a semantic version, like `circleci/hello-build@2.3.1`. Better still, would be to reference an actual working orb, so that if a user were to copy+paste this config, it would be correct.

Reply:
Agreed, we only want to use something that will work after copy+paste. Is here a versioned hello-build orb @ndintenfass ?

Reply:
Yes, you can use 0.0.5

Reply:
Ok, I changed this to hello: circleci/hello-build@0.0.5